Sources/Commands/UserToolchain.swift

```diff
@@ -163,6 +148,34 @@ public struct UserToolchain: Toolchain {
         guard localFileSystem.isExecutableFile(clangCompiler) else {
             throw Error.invalidToolchain(problem: "could not find `clang` at expected path \(clangCompiler.asString)")
         }
+        return clangCompiler
+    }
+
+    public init(destination: Destination) throws {
+        self.destination = destination
+
+        // Get the search paths from PATH.
+        let envSearchPaths = getEnvSearchPaths(
+            pathString: getenv("PATH"), currentWorkingDirectory: localFileSystem.currentWorkingDirectory)
+
+        func lookup(fromEnv: String) -> AbsolutePath? {
```

Comment:
Similarly this could be a private method in this class.

Reply:
done